cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -289,6 +289,26 @@ impl ToString for AluRmiROpcode {
     }
 }
 
+#[derive(Clone, PartialEq)]
+pub enum ReadOnlyGprRmROpcode {
```

Comment:
`ReadOnlyGprRmR*` feels a bit cumbersome to me (and did when I looked at it some days ago).  Would you consider instead `UnaryRmR` (and consistent renaming for the derived-from-it names?)